.travis.yml

```diff
@@ -1,10 +1,5 @@
 language: node_js
```

Comment:
I've been removing Travis completely in favor of GH,  what should we do?

Reply:
Good call.